build/configs/rtl8721csm/tc/defconfig

```diff
@@ -188,8 +188,8 @@ CONFIG_BOOT_RUNFROMFLASH=y
 #
 # Boot Memory Configuration
 #
-CONFIG_RAM_KREGIONx_START="0x0,0x0,"
-CONFIG_RAM_KREGIONx_SIZE="0x0,0x0,"
+CONFIG_RAM_KREGIONx_START="0x10005000,0x02000000,"
+CONFIG_RAM_KREGIONx_SIZE="466944,3145728,"
```

Comment:
Actually this is not what I want. 
This change just sets PSRAM as 3MB regardless of TZ usages. When user disables TZ, user should modify here as well.
But as Realtek said, user could not know that.
The concept of kregion is RAM configuration with physical start address and size. Then, internally TizenRT calculates available region.
The `__ebss` is the mark of the end of system use for first kregion. How about adding something like that to get available region? If we don't, we had better use Realtek way.